### ISSUE TITLE
Plus Editor: Line height not working as expected

### ISSUE THREAD

**Issue by sethmad** [standardnotes/forum#655]
In Plus Editor, the line height tool doesn't work as expected when dealing with multiple paragraphs.  Because the line spacing is styled by the <p> tag, it is currently impossible to have no space between paragraphs.

For example, if line spacing is set to 1, a wrapped string of text will have no spacing between lines as expected. However, a new paragraph will automatically have spacing above. 

In other words, in Plus Editor
you can't have multiple paragraphs
with no spacing between them
like this.

**Comment by mobitar** [comment]
The Plus and Bold editors use `<p>` tags to display paragraphs, and I don't think there's any way for p tags to not have margins, unless you customize the CSS. But that would be outside the scope of the individual editors.

**Comment by sethmad** [comment]
Correct, the `<p>` tag problem was noted in my original post but hidden by a typo. I believe the editor should be modified not to use `<p>` tags, because it limits styling and causes problems as in my original post.

**Comment by Slumberland** [comment]
I have experienced paragraph spacing continuity issues with the Plus editor as well,
I think it may be because I switch between iOS and MacOS on the same note.
I have found that as a (single line paragraph) document progresses, like the OP mentioned above:

In other words, in Plus Editor
you can't have multiple paragraphs
with no spacing between them
like this.

That later in my document the paragraphs jump to double line spacing and I

can no longer create
text blocks like this

If I switch back to Plain Editor it truncates it all.
Is there another editor I can use that will ignore the p tags?

**Comment by baptiste-grob** [comment]
I don't believe we have a rich text editor that doesn't rely on `<p>` tags 🤔 the closest we have to that would be any markdown editor
